### ISSUE TITLE
[$250] Screen Reader: Many Pages: The icon of link opening new page/tab is not announced

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#76941]
## Action Performed:
Prerequisite:
- the user is logged in

1. Using JAWS + Chrome, open the page https://new.expensify.com/settings/profile
2. Navigate using TAB to the 'Aid' and 'What's new' links, in the middle navigation panel
3. Note how the link is displayed and announced

Other Occurrences:
1. On Settings - About: the links View the code and View open jobs
2. On Settings - About - App Download Links: the links of downloads

## Expected Result:
Visually, the link has an informative icon about link opening a new page or tab, the same should also be announced, e.g.:
- 'Aid, opens in new page or tab'
- 'What's new, opens in new page or tab'

## Actual Result:
The informative arrow icon is not announced as part of the link nor announced separately. Screen reader announces:
-"Aid"
-"What’s new"

HTML:
<div aria-label="Aid" role="menuitem" data-tag="pressable" tabindex="0" class="css-g5y9jx r-1otgn73 r-1loqt21 r-18u37iz r-1wtj0ep r-13qz1uu r-1awozwy r-1xfd6ze r-102gzdx r-11f147o r-3pj75a" data-component="GenericPressable" data-element="Pressable" data-source-file="BaseGenericPressable.tsx" style=""><div class="css-g5y9jx r-13awgt0"><div class="css-g5y9jx r-18u37iz"><div class="css-g5y9jx r-eqz5dr r-13awgt0"><div class="css-g5y9jx r-18u37iz r-105ug2t"><div class="css-g5y9jx r-1awozwy r-1777fci r-iyfy8q"><div class="css-g5y9jx" data-testid=""><svg xmlns="[http://www.w3.org/2000/svg"](http://www.w3.org/2000/svg%22) xml:space="preserve" viewBox="0 0 20 20" width="20" height="20" hovered="false" pressed="false" fill="#A2A9A3"><path d="M10 19c5 0 9-4 9-9s-4-9-9-9-9 4-9 9 4 9 9 9m.9-8.1v1.4H8.8v-3h.5c1.3 0 2.1-.7 2.1-1.8s-.6-1.6-1.5-1.6c-.9.1-1.6.9-1.6 1.9H6C6 5.7 7.6 4 9.9 4s3.9 1.2 3.9 3.5c0 1.9-1.2 3.2-2.9 3.4m.3 3.7c0 .8-.6 1.4-1.3 1.4-.8 0-1.3-.6-1.3-1.4 0-.7.6-1.3 1.3-1.3s1.3.6 1.3 1.3" style="fill-rule: evenodd; clip-rule: evenodd;"></path></svg></div></div><div class="css-g5y9jx r-1777fci r-13awgt0" style="min-height: 40px;"><div class="css-g5y9jx r-18u37iz r-1awozwy r-dnmrzs"><div dir="auto" data-selection-scrapper-hidden-element="false" class="css-146c3p1 r-1udh08x r-1udbk01 r-1iln25a" style="color: rgb(0, 46, 34); font-size: 15px; text-align: left; font-family: &quot;Expensify Neue&quot;, &quot;Windows Segoe UI Emoji&quot;, &quot;Noto Color Emoji&quot;, &quot;Custom Emoji Font&quot;; font-style: normal; font-weight: 700; flex-shrink: 1; margin-left: 12px; white-space: pre; direction: ltr; max-width: 100%; line-height: 20px;"><font dir="auto" style="vertical-align: inherit;"><font dir="auto" style="vertical-align: inherit;">Aid</font></font></div></div></div></div></div><div class="css-g5y9jx r-18u37iz" style="min-height: 40px;"><div class="css-g5y9jx r-105ug2t" style="justify-content: center; align-items: center; width: 40px;"><div class="css-g5y9jx" data-testid=""><svg xmlns="[http://www.w3.org/2000/svg"](http://www.w3.org/2000/svg%22) xml:space="preserve" viewBox="0 0 20 20" width="20" height="20" hovered="false" pressed="false" fill="#A2A9A3"><path d="M13.9 4 8.4 9.4c-.6.6-.6 1.5 0 2.1s1.5.6 2.1 0L16 6.1v2.4c0 .8.7 1.5 1.5 1.5S19 9.3 19 8.5V1h-7.5c-.8 0-1.5.7-1.5 1.5S10.7 4 11.5 4z"></path><path d="M1 9.5C1 7.6 2.6 6 4.5 6H7c.8 0 1.5.7 1.5 1.5S7.8 9 7 9H4.5c-.3 0-.5.2-.5.5v6c0 .3.2.5.5.5h6c.3 0 .5-.2.5-.5v-2.4c0-.8.7-1.5 1.5-1.5s1.5.7 1.5 1.5v2.4c0 1.9-1.6 3.5-3.5 3.5h-6C2.6 19 1 17.4 1 15.5z"></path></svg></div></div></div></div></div></div>

## Area issue was found in:
Settings–Profile

## Failed WCAG checkpoints
WCAG 1.1.1 Non-Text Content
https://www.w3.org/WAI/WCAG22/Understanding/non-text-content.html

## User impact:
Users who have difficulty perceiving visual content use assistive technology to read text aloud, present it visually, or convert it to braille. Text alternatives may help some people who have difficulty understanding the meaning of photographs, drawings, and other images (e.g., line drawings, graphic designs, paintings, three-dimensional representations), graphs, charts, animations, etc. People who are deaf-blind can read the text alternatives in braille.

## Suggested resolution:
Add a descriptive aria-label to announce the link together with the icon, e.g.:
- 'Aid, opens in new page or tab'
- 'What's new, opens in new page or tab'
https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA14

## Environment(s)
Windows Windows 10 - Chrome

## Screenshots/Videos
https://github.com/user-attachments/assets/1dc27d42-9af4-45ce-ae5b-6f2e7ebabe83

<img width="1918" height="996" alt="Image" src="https://github.com/user-attachments/assets/a3f9b220-d044-4ce8-955e-56cb3329833f" />

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6994943
**Issue reported by:** NA
**Slack conversation:** NA

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~022014008591570195281</li>
        <li>Upwork Job ID: 2014008591570195281</li>
        <li>Last Price Increase: 2026-01-21</li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @amyevans</details>

**Comment by mallenexpensify** [comment]
### Payment Summary
Contributor: @nkdengineer due $250 via NewDot 
Contributor+: @DylanDylann due $250 via NewDot
